src/wallet/spend.h

```diff
@@ -51,7 +51,10 @@ class COutput
     /** The time of the transaction containing this output */
     int64_t m_time;
 
-    COutput(const CWallet& wallet, const CWalletTx& wtx, int iIn, int depth, bool spendable, bool solvable, bool safe, int64_t time, bool use_max_sig_in = false)
+    /** Whether the transaction containing this output is sent from the owning wallet */
+    bool m_from_me;
+
+    COutput(const CWallet& wallet, const CWalletTx& wtx, int iIn, int depth, bool spendable, bool solvable, bool safe, int64_t time, bool from_me, bool use_max_sig_in = false)
```

Comment:
In commit "wallet: Store whether a COutput is from the wallet" (521a703d056c13d85dbaabd289d2cc7edfd671c0)

Again `use_max_sig_in = false` default argument makes this hard to verify and means it could silently break. This again suggests dropping default argument in earlier commit.